### PR TITLE
Reduce schema disagreements

### PR DESCRIPTION
**Pull request by cdonati**
Sometimes a `CREATE TABLE` statement will time out, and the driver will try to create it again before the cluster fully loads the first one. This branch is meant to make that situation much less likely.

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2242/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2256/

File: AppController/lib/cassandra_helper.rb [hunk 1]
```diff
@@ -79,10 +79,10 @@ def setup_db_config_files(master_ip)
 # Args:
 #   clear_datastore: Remove any pre-existent data in the database.
 #   needed: The number of nodes required for quorum.
-def start_db_master(clear_datastore, needed)
+def start_db_master(clear_datastore, needed, desired)
```
Comment (tmarballi) [inline AppController/lib/cassandra_helper.rb]: Please add the new 'desired' argument in the docs.

File: AppController/lib/cassandra_helper.rb [hunk 2]
```diff
@@ -93,7 +93,7 @@ def start_db_master(clear_datastore, needed)
 # Args:
 #   clear_datastore: Remove any pre-existent data in the database.
 #   needed: The number of nodes required for quorum.
-def start_db_slave(clear_datastore, needed)
+def start_db_slave(clear_datastore, needed, desired)
```
Comment (tmarballi) [inline AppController/lib/cassandra_helper.rb]: Here too, add the new 'desired' argument in the docs.

**Pull request merged**